Demo/Configuration.swift

```diff
@@ -8,7 +8,7 @@ import Adyen
 import Foundation
 import PassKit
 
-internal enum ConfigurationConstants {
+enum ConfigurationConstants {
```

Comment:
This is due to a formatting issue..
Pull latest `develop` and latest `Internal`
Also you need to revert those changes in `Demo/Configuration.swift`